src/components/UI/Card.js

```diff
@@ -1,12 +1,36 @@
-import React from "react";
+import classNames from "classnames";
+import React, {useState} from "react";
 import "./Card.css";
 
-function Card(props) {
+function Card({item}) {
+  // let [classNames, setClassNames] = useState("card");
+  const [isChecked, setChecked] = useState(false);
+
+  const clickHandler = () => {
+    // if (classNames === "card") {
+    //   setClassNames("card card_selected");
+    // } else {
+    //   setClassNames("card");
+    // }
+    setChecked(!isChecked)
+  };
+
   return (
-    <div className="card">
-      <div className="card_header">{props.item.caption}</div>
+    // <div className={classNames}>
+      // <div className={"card" + (isChecked ? " card_selected" : "")}>
+      <div className={(isChecked ? classNames('card', 'card_selected') : classNames('card'))}>
```

Comment:
The power of classnames is not in just joining the classes - we could use join operator for that purpose. The point is in using **conditional styling** instead:
`<div className={classNames({'card': true, 'card_selected': isChecked})}>... </div>`
Read [description/specification](https://www.npmjs.com/package/classnames) to get details.